packages/db/src/schema/status-job.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  datetime,
+  mysqlEnum,
+  mysqlTable,
+  int,
+  varchar,
+  timestamp,
+} from "drizzle-orm/mysql-core";
+import { relations } from "drizzle-orm";
+import { page } from "./page";
+
+export const statusJob = mysqlTable("statusJob", {
```

Comment:
Hearing the name "status job" for the first time. Should we call it "ping"? I have something similar where I called it function `monitor`, but all are related to pinning a website.

Reply:
yep monitor makes sense!